docs/sources/http_api/data_source.md

```diff
@@ -241,6 +242,7 @@ Content-Type: application/json
 
 {"message":"Datasource updated", "id": 1, "name": "test_datasource"}
 ```
+> NOTE: `password` and `basicAuthPassword` should be defined under `secureJsonData` in order to be stored in an encrypted blob in the database and no more sent to the browser. Check also the [Encrypting Sensitive Data]({{< relref "../plugins/developing/auth-for-datasources.md/#encrypting-sensitive-data">}}) documentation for more details.
```

Comment:
I guess the example above should be updated, since it doesn't use `secureJsonData` and also includes `password` and `basicAuthPassword` directly?